csharp/src/Ice/Connection.cs

```diff
@@ -2,6 +2,8 @@
 
 #nullable enable
 
+using Ice.Internal;
```

Comment:
Why do we need this `using` now?
We didn't have it before, and it doesn't seem like any internal types were added here.

Reply:
Removed. It was not needed.